apps/docs/pages/guides/platform/shared-responsibilty-model.mdx

```diff
@@ -0,0 +1,49 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'shared-responsibility-model',
+  title: 'Shared Responsibility Model',
+  description: 'Running databases is a shared responsibility between you and Supabase.',
+}
+
+Running databases is a shared responsibility between you and Supabase. There are some things that we can take care of for you, and some things that you are responsible for. This is by design: we to give you the freedom to use your database however you want. While we _could_ put many more restrictions in place to ensure that you can’t do anything wrong, you will eventually find those restrictions prohibitive.
+
+Generally, we aim reduce your burden of managing infrastructure and knowing about Postgres internals, minimizing configuration as much as we can. Here are a few things that you should know:
+
+## You share the security responsibility
+
+We give you full access to the database. If you share that access with other people (either people on your team, or the public in general) then it is your responsibility to ensure that the access levels you provide are correctly managed.
+
+If you have an inexperienced member on your team, then you probably shouldn’t give them access to Production. You should set internal workflows around what they should and should not be able to do, with restricted access to avoid anything that might be deemed dangerous.
+
+You are also responsible for ensuring that tables with sensitive data has the right level of access. You are also responsible for managing your database secrets and API keys, storing them safely in an encrypted store.
+
+## You can crash your database
+
+”Crash” is used liberally here. It refers to any situation where your application is unable to work as desired because of DB issues. It could be because the DB is unable to support the number of connections being attempted, or because its performance is unacceptably low because of CPU or memory constraints.
+
+You are responsible of provisioning enough compute to run the workload that your application requires. You are also responsible for employing methods to manage the workload: adding indexes, adding filters on large queries, caching strategies, materialized views, etc.
```

Comment:
Worth mentioning that we provide reporting tools to help them make the right decisions on CPU/memory/etc?
https://app.supabase.com/project/_/reports/database